gtsam/navigation/AHRSFactor.h

```diff
@@ -182,24 +181,26 @@ class GTSAM_EXPORT AHRSFactor: public NoiseModelFactor3<Rot3, Rot3, Vector3> {
 
   /// predicted states from IMU
   /// TODO(frank): relationship with PIM predict ??
-  static Rot3 Predict(
+  static Rot3 Predict(const Rot3& rot_i, const Vector3& bias,
+                      const PreintegratedAhrsMeasurements& pim);
+
+  /// @deprecated constructor, but used in tests.
+  AHRSFactor(Key rot_i, Key rot_j, Key bias,
+             const PreintegratedAhrsMeasurements& pim,
+             const Vector3& omegaCoriolis,
+             const boost::optional<Pose3>& body_P_sensor = boost::none);
+
+  /// @deprecated static function, but used in tests.
+  static Rot3 predict(
       const Rot3& rot_i, const Vector3& bias,
-      const PreintegratedAhrsMeasurements preintegratedMeasurements);
+      const PreintegratedAhrsMeasurements& pim, const Vector3& omegaCoriolis,
+      const boost::optional<Pose3>& body_P_sensor = boost::none);
 
+#ifdef GTSAM_ALLOW_DEPRECATED_SINCE_V42
   /// @deprecated name
   typedef PreintegratedAhrsMeasurements PreintegratedMeasurements;
```

Comment:
I think `GTSAM_DEPRECATED` can be used, but not very sure if that would compile

Reply:
Leaving as is